content/docs/for-buildpack-authors/concepts/layer.md

```diff
@@ -3,8 +3,12 @@ title="What is a buildpack dependency layer?"
 weight=99
 +++
 
+`Dependency layers` are semantically meaningful layers that are contributed by one or more buildpacks during the building phase, one for each dependency.
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+![builder](/images/builder.svg)
+
+As seen in the image above, buildpacks read application source code and build dependency layers. Multiple buildpacks can contribute a subset of an app's required dependencies that are added as subdirectories under the `CNB_LAYERS_DIR` directory. These dependencies then get exported as layers in the final app image or build cache.
 
-If you are familiar with this content and would like to make a contribution, please feel free to open a PR :)
+The `build` phase runs the build binary of each buildpack that outputs zero or more layers into `$(CNB_LAYERS_DIR)/<buildpack-id>` and writes metadata for each layer as TOML files in that directory. During the `export` phase, all layers created by the buildpacks are either cached or added to the output application image.
```

Comment:
Small nit, sometimes we say TOML and sometimes we say `toml`. No preference as to which we choose but we should be consistent where possible.

Reply:
updated all to toml